### PR TITLE
Fix: Monaco 404 errors

### LINKED ISSUE #3221: 404 errors on console from monaco searching for web workers
**Describe the bug**
The editor and json web workers can't be found and as a result a lot of 404 errors are being returned

**To Reproduce**
1. Go to 'https://developer.microsoft.com/en-us/graph/graph-explorer'
2. Open developer tools
3. Open the console tab
4. See errors

**Expected behavior**
Errors shouldn't be present

**Screenshots**
image

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] Windows
 - Browser [e.g. chrome, safari] Edge

**Smartphone (please complete the following information):**

**Additional context**

### PR DESCRIPTION
**Pull request by ElinorW**
## Overview
Fix #3221 
updates code to fetch web workers from the correct destination

### Demo
Updated 
image

### Notes
The worker URL functions previously existed but we removed them as we had stopped using the monaco-editor webpack plugin. Once we bundled Monaco, we resumed using the plugin and must have forgotten to update the worker urls. 
Ref: https://github.com/microsoftgraph/microsoft-graph-explorer-v4/pull/2615

## Testing Instructions

* Checkout this branch
* Load GE
* Check network tab on developer tools
* Observe 200 OK on editor.worker.js/json.worker.js calls

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -160,6 +160,32 @@ function loadResources() {
...
 
+/**
+ * Set's up Monaco Editor's Workers.
+ */
+enum Workers {
+  Json = 'json',
+  Editor = 'editor',
+}
+
+(window as any).MonacoEnvironment = {
```
Comment (musale) [inline src/index.tsx:171]:
```suggestion
import { Environment } from 'monaco-editor/esm/vs/editor/editor.api';
// ... other imports

declare global {
  interface Window {
    MonacoEnvironment: Environment;
  }
}

window.MonacoEnvironment = {
```

To reduce using `any`s, assign the `MonacoEnvironment` on the `window`.

File: src/index.tsx [hunk 2]
```diff
@@ -160,6 +160,32 @@ function loadResources() {
...
+  getWorkerUrl(moduleId: any, label: string) {
+    if (label === 'json') {
+      return getWorkerFor(Workers.Json);
+    }
+    return getWorkerFor(Workers.Editor);
+  }
```
Comment (musale) [inline src/index.tsx:177]: We have all the other workers too, do we need to care about them? The default will be editor.worker.js.
image

**Review by musale — DISMISSED** [review]
Good to go 🚢

**Review by thewahome — DISMISSED** [review]

File: src/index.tsx (hunk 2 above)
Comment (ElinorW) [reply src/index.tsx:186]: yeah, since they exist I will add them

**Review by thewahome — APPROVED** [review]

**Pull request merged**